pilot/model/validation.go

```diff
@@ -877,9 +877,11 @@ func ValidateEgressRule(msg proto.Message) error {
 			errs = multierror.Append(errs, err)
 		}
 
-		if cidrDestinationService && Protocol(strings.ToUpper(port.Protocol)) != ProtocolTCP {
-			errs = multierror.Append(errs, fmt.Errorf("Only TCP protocol can be defined for CIDR destination "+
-				"service notation. port: %d protocol: %s destination.service: %s",
+		if cidrDestinationService &&
+			!IsEgressRulesSupportedTCPProtocol(Protocol(strings.ToUpper(port.Protocol))) {
+			errs = multierror.Append(errs, fmt.Errorf("Only the following protocols can be defined for "+
+				"CIDR destination service notation "+egressRulesSupportedTCPProtocols()+
```

Comment:
nit: use string formatting to build the entire string instead of a mix of string concatenation and string formatting?